packages/forms/src/Components/Repeater.php

```diff
@@ -440,6 +449,15 @@ public function getCachedExistingRecords(): Collection
         );
     }
 
+    public function getItemLabel(string $uuid): ?string
+    {
+        $itemData = data_get($this->getLivewire(), "{$this->getStatePath()}.{$uuid}");
```

Comment:
I think I'd prefer just `$item`, but let's see what the others think.

Reply:
Yup